lib/config-seed/version

```diff
@@ -1 +1 @@
-3.2.2
+3.0.0
```

Comment:
```suggestion
3.3.0
```
:wink:

Reply:
already done, noticed right after creating it 🤣